Core/Core/Profile/ProfileViewController.swift

```diff
@@ -108,6 +101,16 @@ public class ProfileViewController: UIViewController, ProfileViewProtocol {
 
     public func reload() {
         tableView?.reloadData()
+        reloadProfile()
+    }
+
+    func reloadProfile() {
+        let profile = presenter?.profile.first
+        let userName = presenter?.profile.first?.name ?? env.currentSession?.userName
+        avatarView?.name = userName ?? ""
+        avatarView?.url = profile?.avatarURL
+        nameLabel?.text = userName.flatMap { User.displayName($0, pronouns: profile?.pronouns) }
```

Comment:
do we really need pronouns in the "my profile"? I thought they are mostly for helping you politely refer to other people.